src/target/llvm/codegen_llvm.cc

```diff
@@ -594,6 +596,7 @@ llvm::Type* CodeGenLLVM::DTypeToLLVMType(const DataType& dtype) const {
     return llvm::VectorType::get(etype, dtype.lanes());
 #endif
   } else {
+    ICHECK(etype != nullptr) << "No suitable llvm type found for dtype: " << dtype;
```

Comment:
this breaks CI

Reply:
Thank you @vinx13. I was trying to leave the code better than I found it, but I guess some downstream handling is relying on this behavior. I have removed the check.